{"uuid": "e102699d-a407-490a-b5db-67c5e3ffc89b", "children": ["3b938997-ca76-448d-aa17-866e96ed768d", "3282dfc9-dbca-42ef-9773-84fce6564a3a", "31176f3f-dad6-42e5-89e1-4d389347eab3", "c9826129-e12c-4ced-97fd-cd520e6381d2"], "befores": [{"name": "initialize_browser_state", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: Page.goto: net::ERR_CONNECTION_CLOSED at https://nikita-filonov.github.io/qa-automation-engineer-ui-course/#/auth/registration\nCall log:\n  - navigating to \"https://nikita-filonov.github.io/qa-automation-engineer-ui-course/#/auth/registration\", waiting until \"networkidle\"\n\n", "trace": "  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\fixtures\\browsers.py\", line 24, in initialize_browser_state\n    registration_page.visit('https://nikita-filonov.github.io/qa-automation-engineer-ui-course/#/auth/registration')\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\pages\\base_page.py\", line 12, in visit\n    self.page.goto(url, wait_until='networkidle')\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 9019, in goto\n    self._sync(\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\_impl\\_page.py\", line 556, in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\_impl\\_frame.py\", line 153, in goto\n    await self._channel.send(\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 69, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 558, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n"}, "steps": [{"name": "Opening the url \"https://nikita-filonov.github.io/qa-automation-engineer-ui-course/#/auth/registration\"", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: Page.goto: net::ERR_CONNECTION_CLOSED at https://nikita-filonov.github.io/qa-automation-engineer-ui-course/#/auth/registration\nCall log:\n  - navigating to \"https://nikita-filonov.github.io/qa-automation-engineer-ui-course/#/auth/registration\", waiting until \"networkidle\"\n\n", "trace": "  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\pages\\base_page.py\", line 12, in visit\n    self.page.goto(url, wait_until='networkidle')\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 9019, in goto\n    self._sync(\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\_impl\\_page.py\", line 556, in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\_impl\\_frame.py\", line 153, in goto\n    await self._channel.send(\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 69, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dastan\\PycharmProjects\\autotests-ui\\.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 558, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n"}, "start": 1754484277359, "stop": 1754484278408}], "start": 1754484276651, "stop": 1754484278409}], "start": 1754484276651, "stop": 1754484280144}